app/src/main/java/org/covital/measurements/presentation/MeasurementResultFragment.kt

```diff
@@ -1,32 +1,24 @@
 package org.covital.measurements.presentation
 
-import android.os.Bundle
-import android.view.LayoutInflater
-import android.view.View
-import android.view.ViewGroup
-import androidx.databinding.DataBindingUtil
-import androidx.fragment.app.Fragment
 import org.covital.R
+import org.covital.common.presentation.BaseFragment
 import org.covital.databinding.FragmentMeasurementResultBinding
+import org.covital.sharedGraphViewModel
 import org.koin.androidx.viewmodel.ext.android.sharedViewModel
 
-class MeasurementResultFragment : Fragment() {
-    private val sharedViewModel: MeasurementsViewModel by sharedViewModel()
+class MeasurementResultFragment : BaseFragment<FragmentMeasurementResultBinding>() {
+
+    override val layoutRes: Int = R.layout.fragment_measurement_result
+
+    private val shared: MeasurementsViewModel by sharedGraphViewModel(R.id.measure_graph)
```

Comment:
I prefer keeping the name sharedViewModel, or sharedVm. I find shared does not have enough meaning on its own. What do you think?

Reply:
Sure - `sharedViewModel` was giving me issues because of extension functions of that name